### PR TITLE
Bug fix : Wrong declaration

### PR DESCRIPTION
**Pull request by jtavva**
Fix for #375

### CODE REVIEW

File: ColorMyViews/app/src/main/java/com/android/example/colormyviews/MainActivity.kt [hunk 1]
```diff
@@ -49,9 +50,9 @@ class MainActivity : AppCompatActivity() {
```
Comment (kkuan2011) [inline ColorMyViews/app/src/main/java/com/android/example/colormyviews/MainActivity.kt:50]: Should the above findByViewId calls be with TextView or Button?

**Review by kkuan2011 — APPROVED** [review]

**Review by kkuan2011 — APPROVED** [review]

**Comment by kkuan2011** [comment]
Oh I also have one comment on the earlier CL and I think the branch has a conflict

File: ColorMyViews/app/src/main/java/com/android/example/colormyviews/MainActivity.kt (hunk 1 above)
Comment (jtavva) [reply ColorMyViews/app/src/main/java/com/android/example/colormyviews/MainActivity.kt:50]: No they are Textviews.
<TextView
        android:id="@+id/box_one_text"

**Comment by jtavva** [comment]
> Oh I also have one comment on the earlier CL and I think the branch has a conflict

Yes I resolved it.

**Pull request merged**